### PR TITLE
Add luhn exercise

### PR DESCRIPTION
**Pull request by pfertyk**

### CODE REVIEW

File: exercises/practice/luhn/.meta/config.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
+{
+  "authors": [],
```
Comment (BNAndras) [inline exercises/practice/luhn/.meta/config.json:2]: Needs your GitHub handle here.

File: exercises/practice/luhn/luhn_test.gd [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+
+func test_non_numeric_non_space_char_in_the_middle_with_a_sum_that_s_divisible_by_10_isn_t_allowed(solution_script):
+	solution_script.card_num = "59%59"
+	return [solution_script.valid(), false]
+
+
+func test_is_valid_can_be_called_repeatedly(solution_script):
+	solution_script.card_num = "055 444 285"
+	return [solution_script.valid(), true]
+	return [solution_script.valid(), true]
```
Comment (BNAndras) [inline exercises/practice/luhn/luhn_test.gd:114]:
```suggestion
```

Extra return line

**Review by BNAndras — APPROVED** [review]

File: exercises/practice/luhn/.meta/config.json (hunk 1 above)
Comment (pfertyk) [reply exercises/practice/luhn/.meta/config.json]: Thanks, added! :)

File: exercises/practice/luhn/luhn_test.gd (hunk 2 above)
Comment (pfertyk) [reply exercises/practice/luhn/luhn_test.gd:117]: This is actually intentional, the rationale is given in the corresponding Python test file. I copied the comment from there, to make it clear ;)

**Comment by pfertyk** [comment]
@BNAndras in this PR I'm also making a small change in the `flatten-array` file, I didn't notice before that it uses spaces instead of tabs. I hope you don't mind ;)

**Comment by BNAndras** [comment]
> @BNAndras in this PR I'm also making a small change in the `flatten-array` file, I didn't notice before that it uses spaces instead of tabs. I hope you don't mind ;)

I’d probably make it a separate PR for consistency, but leaving the unrelated change in is fine for now.

**Pull request merged**